src/components/ToyForm.js

```diff
@@ -8,12 +8,12 @@ const ToyForm = ({ toy }) => {
   const [form, setForm] = useState({
     name: '',
     description: '',
-    image_url: '',
+    imageUrl: '',
   });
 
   const handleChange = (event) => {
     setForm({
-      ...form,
+      form,
```

Comment:
`form` initializes as:
```
{ name: '', description: '', imageUrl: '' }
```

When handleChange hits (assuming I typed "My Name" into name field), it's going to set form to this:
```
{ form: { name: '', description: '', imageUrl: '' }, name: 'My name' }
```
Then, if I change "My name" to "Your name", it's going to look like this:
```
{ form: { form: { name: '', description: '', imageUrl: ''}, name: 'My name' }, name: 'Your name' }
```

Reply:
https://jsfiddle.net/zg4nu6ok/